detection_rules/docs.py

```diff
@@ -598,13 +597,13 @@ def metadata_str(self) -> str:
 
     def guide_str(self) -> str:
         """Add the guide section to the rule detail page."""
-        guide = re.sub(self.elastic_hyperlink_pattern, r'\1', self.rule['note'])
-        return f'{AsciiDoc.title(4, "Investigation guide")}\n\n\n{AsciiDoc.code(guide, code="markdown")}'
+        guide = convert_markdown_to_asciidoc(self.rule['note'])
+        return f'{AsciiDoc.title(4, "Investigation guide")}\n\n\n{AsciiDoc.content(guide)}'
```

Comment:
Do we need this `content` method. It doesn't appear to do anything.

```suggestion
        return f'{AsciiDoc.title(4, "Investigation guide")}\n\n\n{guide}'
```

Reply:
Implemented suggestion.